### PR TITLE
Pods 2823 H&S sch reg - Routes and blank controllers/ids for establisher company director journey

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: conf/establishersCompanyDirector.routes [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+GET        /:establisherIndex/company/director/:directorIndex/director-date-of-birth                                controllers.register.establishers.company.director.DirectorDOBController.onPageLoad(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/director-date-of-birth                                controllers.register.establishers.company.director.DirectorDOBController.onSubmit(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+GET        /:establisherIndex/company/director/:directorIndex/change/director-date-of-birth                         controllers.register.establishers.company.director.DirectorDOBController.onPageLoad(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/change/director-date-of-birth                         controllers.register.establishers.company.director.DirectorDOBController.onSubmit(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
```
Comment (madhuranakate) [inline conf/establishersCompanyDirector.routes]: DirectorDOB routes are needed in UpdateMode

File: conf/establishersCompanyDirector.routes [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+POST       /:establisherIndex/company/director/:directorIndex/reason-for-no-national-insurance-number               controllers.register.establishers.company.director.DirectorNoNINOReasonController.onSubmit(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+GET        /:establisherIndex/company/director/:directorIndex/change/reason-for-no-national-insurance-number        controllers.register.establishers.company.director.DirectorNoNINOReasonController.onPageLoad(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/change/reason-for-no-national-insurance-number        controllers.register.establishers.company.director.DirectorNoNINOReasonController.onSubmit(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
```
Comment (madhuranakate) [inline conf/establishersCompanyDirector.routes]: DirectorNoNinReason routes are needed in UpdateMode

File: conf/establishersCompanyDirector.routes (hunk 1 above)
Comment (grantrobson) [reply conf/establishersCompanyDirector.routes]: done

File: conf/establishersCompanyDirector.routes (hunk 2 above)
Comment (grantrobson) [reply conf/establishersCompanyDirector.routes]: done

**Pull request merged**